### DISCUSSION TITLE
myVaillant app isn't working "Gateway unreachable"

### DISCUSSION THREAD

**Discussion by zolakt** [signalkraft/mypyllant-component#98] [General]
**NOTE: This isn't a problem with the mypyllant integration, but with the myVaillant app itself.**

I know this shouldn't be posted here, and I apologise, but given the lack of any sort of community from Vaillants side, I'm hoping someone here has the same problem, and a suggestion on what to do. I've emailed Vaillant, but haven't got a response yet.

In the last two days (maybe more) the myVaillant app is not working. It throw this error "Gateway unreachable. Have you changed your Wi-Fi password or settings? Is the gateway connected to an outlet and switched on?"

However, everything is working fine in mypyllant.

I have the VR921 module (connected with ethernet cable). I've tried unplugging it from power and plugging it back in. The myVaillant app throws the same exception, while mypyllant connects after a minute or so, and works just fine.

Is anyone having the same issue?

Btw. congratulations to everyone involved with mypyllant. This is definitive proof that you guys have done a much better job than Vaillant themselves 😃👍

**Comment by LAScotland** [comment]
Yep - same here. I'm assuming the app is borked - for now, i've recreated it in a HA dashboard, and given my wife access to that. TBH is probably better that way:

![image](https://github.com/signalkraft/mypyllant-component/assets/95568985/ebf73b59-0966-4926-a516-9a43b8e8dd58)

**Comment by zolakt** [comment]
Ahh, glad to know I'm not the only one, at least.
I have pretty much everything I need in HA, but still you can't do everything, like edit the schedule.

**Comment by Honzik1234** [comment]
Log out, uninstall the app and you'll be fine.
I service Vaillant and it works for customers.

**Comment by signalkraft** [comment]
> I have pretty much everything I need in HA, but still you can't do everything, like edit the schedule.

You should be able to edit the schedule in the HA calendar, although it may depend on what type of controller you have. It's untested on VRC700.

**Comment by ear9mrn** [comment]
Looks like a new version of the app is out. For me on Android v2.13.2 (20406) and iphone v2.13.2 (20349). A few days ago the app stopped working (could not connect) but HA was fine. I then realised there was a new version of the app waiting to be installed, so i did, and I was then able to connect again. I am guessing they updated the sensonet first which broke the apps ability to connect.  However, now HA cannot connect and I have no data or working thermostat.

**Comment by signalkraft** [comment]
FYI, I turned this into a discussion, to keep it separate from issues affecting the component itself.

**Comment by zolakt** [comment]
It fixed itself. Probably it installed the new version, as @ear9mrn said.
I'm closing this ticket. Sorry again for spamming this repo, but didn't know where else to turn.
